timescaledb/getting-started/add-data.md

```diff
@@ -5,25 +5,23 @@ stock trade data (also known as "tick data") from [twelvedata][twelve-data].
 
 ## About the dataset
 
-The dataset contains second-by second stock-trade data for the top 100 most-traded symbols, in a hypertable named `stocks_real_time`. It also
-includes a separate table of company symbols and company names, in a table named `company`. `company` is not a hypertable.
+The dataset contains second-by second stock-trade data for the top 100 most-traded symbols, in a 
+hypertable named `stocks_real_time`. It also includes a separate table of company symbols and company 
+names, in a table named `company`. `company` is not a hypertable.
 
 ### Table details
-`stocks_real_time`: Stock data. Includes stock price quotes at every second during trading hours.
 
-```bash
+`stocks_real_time`: Stock data. Includes stock price quotes at every second during trading hours.
     * time (timestamptz): timestamp column incrementing second by second
     * symbol (text): symbols representing a company, mapped to company names in the `company` table
     * price (double precision): stock quote price for a company at the given timestamp
     * day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```

Comment:
```suggestion
* time (timestamptz): timestamp column incrementing second by second
* symbol (text): symbols representing a company, mapped to company names in the `company` table
* price (double precision): stock quote price for a company at the given timestamp
* day_volume (int): number of shares traded each day, NULL values indicate the market is closed
```

Assuming this isn't intended to be a code block anymore? One little annoying thing about Markdown: if they're used outside of a nested list, indents automatically create code blocks.